target/linux/ath79/dts/qca9563_tplink_tl-wpa8630p-v2.dtsi

```diff
@@ -1,45 +1,12 @@
 // SPDX-License-Identifier: GPL-2.0-or-later OR MIT
 
-#include <dt-bindings/gpio/gpio.h>
-#include <dt-bindings/input/input.h>
-
-#include "qca956x.dtsi"
+#include "qca9563_tplink_tl-wpa8630.dtsi"
 
 / {
 	aliases {
-		led-boot = &led_power;
-		led-failsafe = &led_power;
-		led-running = &led_power;
-		led-upgrade = &led_power;
 		label-mac-device = &eth0;
 	};
 
-	leds {
-		compatible = "gpio-leds";
-
-		led_power: power {
-			label = "tp-link:green:power";
-			gpios = <&gpio 1 GPIO_ACTIVE_LOW>;
-		};
-
-		wifi2g {
-			label = "tp-link:green:wifi2g";
-			gpios = <&gpio 19 GPIO_ACTIVE_LOW>;
-			linux,default-trigger = "phy1tpt";
-		};
-
-		wifi5g {
-			label = "tp-link:green:wifi5g";
-			gpios = <&gpio 21 GPIO_ACTIVE_LOW>;
-			linux,default-trigger = "phy0tpt";
-		};
-
-		lan {
-			label = "tp-link:green:lan";
-			gpios = <&gpio 5 GPIO_ACTIVE_LOW>;
-		};
-	};
-
 	gpio-export {
```

Comment:
Is this LED Control GPIO common to both V1 and V2 devices? I have no idea. It should be easy to check on V1 device with:

```
# Setup
echo 14 > /sys/class/gpio/export
echo out > /sys/class/gpio/gpio14/direction
echo 0 > /sys/class/gpio/gpio14/active_low
```
```
# Test
echo 0 > /sys/class/gpio/gpio14/value
echo 1 > /sys/class/gpio/gpio14/value
```

@finasfinas could you try on your V1? If GPIO 14 is something other than the LEDs on your device, a reboot should clear the effects of any of this testing.

Reply:
Until 31 of August I only have remote access to the device. I ran the commands remotely and no error output.
On 31 August, when I have physical access to the device again, I will check if they do something to the leds and report back.

Reply:
According to the V1 source, it is identical to the V2 LED control GPIO, so I put it in the patch here: https://github.com/openwrt/openwrt/pull/3325 . It will still be nice to verify!